README.md

```diff
@@ -14,9 +14,7 @@ For a local demo or test installation, [see below](#development--demo-installati
 ### Requirements
 
 - Collabora Online server installed and running.
-- Drupal 10 (tested on 10.1), maybe compatible with 9.
-- JWT and Media are set as dependencies for the module and are
-  necessary.
+- Composer packages and Drupal modules as specified in `composer.json` and `collabora_online.info.yml`.
```

Comment:
Actually we are even now not mentioning the submodule dependencies.
My concern here was that at some point we stop updating the dependencies in the README.md, and it will get out of date.
E.g. we will need to change it again in the 11.x update.

Anyway, I pushed a commit that restores the explicit dependencies.